src-env (mixdbvendors)/_MicroserviceTemplate_.WebApi/Startup.cs

```diff
@@ -61,11 +63,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
             services.AddCors();
 
-            #if SQLSERVER
             ConfigureSqlServerContext(services);
```

Comment:
Change the method name to ConfigureDatabaseProviderContext

Reply:
Done